### ISSUE TITLE
Error message on login

### ISSUE THREAD

**Issue by lhk** [spotify/android-auth#34]
Hi,

we are using your library for a small university project.
So far, it worked like a charm.
Today our automated tests started failing.
Now, both at runtime and during the tests, the login creates these error messages:
![grafik](https://user-images.githubusercontent.com/1475537/33444334-4ac96360-d5fa-11e7-9a70-0f6b1ffd347c.png)

It is very hard for us to debug this, we can't step through the html/javascript code.
Basically, up to opening the webview, everything seems to work fine.

After entering login credentials, this event is called with a -1 result code:
https://github.com/spotify/android-auth/blob/4be7da15c64dd59bd83e4e1b549a48c5bb09167e/auth-lib/src/main/java/com/spotify/sdk/android/authentication/AuthenticationClient.java#L340

We tried different users, the problem persists.

Do you know how to approach this ?
